.github/workflows/pull-request.yml

```
@@ -312,3 +312,21 @@ jobs:
+      - name: Create .env file for talawa api testing environment
+        run: cp envFiles/.env.devcontainer .env
+
+      - name: Build talawa api non production environment docker image
+        run: docker buildx build --file ./docker/api.Containerfile --tag talawa_api --target non_production ./
+      - name: Start Talawa API Container
+        run: docker compose up -d
+      - name: Import Sample Data
+        run: |
+          docker exec talawa-api-1 /bin/bash -c "pnpm run import:sample-data"
+      - name: Stop Talawa API Container
+        run: docker compose down
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)